src/components/actions/ActionBar.tsx

```diff
@@ -35,6 +39,12 @@ export interface IActionBarProps extends IActionBarOwnProps, IActionBarStateProp
   IActionBarChildrenProps { }
 
 export class ActionBar extends React.Component<IActionBarProps, any> {
+  static DEFAULT_ACTIONS_CONTAINER_CLASSES = [
```

Comment:
Minor detail but shouldn't this be private?

Reply:
done thx